xbmc/system.h

```diff
@@ -115,26 +115,15 @@
 #define HAS_ZEROCONF
 #define HAS_AVAHI
 #endif
-#ifdef HAVE_DBUS
-#define HAS_DBUS
-#endif
 #define HAS_GL
 #ifdef HAVE_SDL
 #define HAS_SDL
-#else
-#ifndef HAVE_X11
-#define HAS_LINUX_EVENTS
-#endif
 #endif
 #define HAS_LINUX_NETWORK
-#ifdef HAVE_LIRC
-#define HAS_LIRC
-#endif
-#endif
-
 #ifdef HAVE_LIBSSH
 #define HAS_FILESYSTEM_SFTP
 #endif
+#endif
```

Comment:
now it is readded at the wrong place

Reply:
seems ok to me?

Reply:
`HAS_FILESYSTEM_SFTP` is now inside `#if defined(TARGET_LINUX) || defined(TARGET_FREEBSD)` which it wasn't before and also changes behaviour.